### PR TITLE
Fix template for tree view

### PR DESCRIPTION
**Pull request by qsomazzi**
## Changelog

```markdown
### Fixed
- Category tree view wasn't easily customizable

```

## Subject

The goal of this PR is too make the category tree view template customizable

### CODE REVIEW

**Comment by greg0ire** [comment]
No docs?

**Comment by qsomazzi** [comment]
@greg0ire @OskarStark doc added

File: Resources/doc/reference/advanced_usage.rst [hunk 1]
```diff
@@ -0,0 +1,21 @@
+.. index::
+    single: Configuration
+
+Advanced usage
+==============
+
+Override Tree template
+----------------------
+
+By default, the ``SonataClassificationBundle`` use his own view template for the admin tree of categories.
```
Comment (greg0ire) [inline Resources/doc/reference/advanced_usage.rst]: "use his" =>"uses its"

File: Resources/doc/reference/advanced_usage.rst [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+You can easily customize it with ``SonataAdminBundle`` configuration :
```
Comment (greg0ire) [inline Resources/doc/reference/advanced_usage.rst]: "with" => "with the"

**Comment by core23** [comment]
Do we need docs for this? Isn't it part of the general admin docs?

**Comment by qsomazzi** [comment]
@core23 that's what I thought, but indeed it's not written anywhere, so it doesn't hurt

**Comment by greg0ire** [comment]
Yeah plus what I really wanted to have documented here is the fact that you can customize `tree`. It's good to have a list of templates that can be customized.

**Comment by qsomazzi** [comment]
@greg0ire all good now ?

**Comment by qsomazzi** [comment]
Thanks all !

**Comment by greg0ire** [comment]
Yes sorry… thanks @qsomazzi

**Pull request merged**